primitives/npos-elections/src/lib.rs

```diff
@@ -418,49 +494,22 @@ pub trait EvaluateSupport {
 
 impl<AccountId: IdentifierT> EvaluateSupport for Supports<AccountId> {
 	fn evaluate(&self) -> ElectionScore {
-		let mut min_support = ExtendedBalance::max_value();
-		let mut sum: ExtendedBalance = Zero::zero();
+		let mut minimal_stake = ExtendedBalance::max_value();
+		let mut sum_stake: ExtendedBalance = Zero::zero();
 		// NOTE: The third element might saturate but fine for now since this will run on-chain and
```

Comment:
The saturation here is fine since we want to minimize it anyway?

Reply:
mainly because we also don't care at this point and it is not the least significant element of the score.